dan_layer/engine_types/src/fees.rs

```diff
@@ -14,10 +14,13 @@ use crate::resource_container::ResourceContainer;
 #[cfg_attr(feature = "ts", derive(TS), ts(export, export_to = "../../bindings/src/types/"))]
```

Comment:
To fix where the bindings are output you need to add another `../`
```suggestion
#[cfg_attr(feature = "ts", derive(TS), ts(export, export_to = "../../../bindings/src/types/"))]
```

Reply:
if I do this the includes in the generated files will be wrong. That's why I move them, instead of correcting the paths in the generated files.